game/lib/controls.py

```diff
@@ -112,7 +112,10 @@ def loop(self):
             pygame.draw.rect(self.__game.get_screen(), GREEN, (self.__exit_x, self.__exit_y,
                                                                self.__exit_x_length, self.__exit_y_length), 2)
             if click[0]:
```

Comment:
Nie wiem czy to twoje, ale to 0 wpisać do zmiennej i nazwać żeby było wiadomo co to za kliknięcie. To chyba jest primary_mouse_button?